core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.ts

```diff
@@ -65,12 +131,12 @@ export class DotToolbarUserComponent implements OnInit {
      * @param any $event
      * @memberof To/olbarUserComponent
      */
-    logoutAs($event): void {
+    logoutAs($event: Event): void {
         $event.preventDefault();
 
         this.loginService.logoutAs().subscribe(
```

Comment:
We need to add a take(1) here.
```suggestion
        this.loginService.logoutAs().pipe(take(1)).subscribe(
```

Reply:
Alright!